avro-fastserde/build.gradle

```diff
@@ -31,7 +31,8 @@ if (USE_AVRO_14) {
 }
 
 dependencies {
-  compile project(":avro-migration-helper")
+  compile project(":helper:helper")
+  compile project(":helper:impls:helper-impl-14")
```

Comment:
why is this required?

Reply:
Need this function `SchemaNormalization#fingerprint64` from ":helper:impls:helper-impl-14".

Reply:
i think its better to pull up SchemaNormalization into something like com.linkedin.avroutil1.compatibility.SchemaNormalization in the helper module, and consider that API

Reply:
Done. Please check the latest commit. Thanks!

Reply:
so is this line still there? github shows it is

Reply:
Good catch! This line is not needed any more. Removed.